Lib/json/tool.py

```diff
@@ -72,4 +72,7 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    try:
+        main()
+    except BrokenPipeError as e:
```

Comment:
nitpick: i prefer to avoid single letter variable name, I like "exc" for the except variable.